### ISSUE TITLE
[Bug]: missing option to change timezone in Linux build 11.10.2

### ISSUE THREAD

**Issue by Jeefongithub** [Floorp-Projects/Floorp#945]
### Pre-Submission Checklist

- [X] I have checked the existing/closed issues to ensure this issue has not already been reported.
- [X] I am using the latest supported version of the browser.
- [X] I have tried disabling all browser extensions/add-ons to rule out interference from these.
- [X] I have tried clearing the browser cache and cookies.
- [X] I have tried to create a new profile and the issue still occurs.
- [X] I have provided detailed step-by-step instructions on how to reproduce the issue.
- [X] I have included relevant screenshots or console outputs.
- [X] I have checked this problem is not ocurring in the latest version of Firefox ESR.
- [X] This issue is specific to this browser and does not occur in other browsers.

### Bug Summary

Timezone be default is set to UTC and there is not option to change from within settings. Some users have an option it appears in the general tab by language to use operating system time but I don't have that option

### Reproduction Steps

1. Browse web for sites using time stamps and verify using UTC time
2. Goto settings in general tab. No option to use OS time

### Expected Behavior

Have option similar to this screenshot taken from floorp subreddit

![image](https://github.com/Floorp-Projects/Floorp/assets/53583661/c9aaeac2-9d0e-4144-b357-cf7656381a5f)

### Actual Behavior

![image](https://github.com/Floorp-Projects/Floorp/assets/53583661/d6e4bff4-39e8-48e6-8135-54f14aa80d7b)

### Screenshots

![image](https://github.com/Floorp-Projects/Floorp/assets/53583661/d6e4bff4-39e8-48e6-8135-54f14aa80d7b)

### Browser Version

11.10.2

### Operating System

Debian Sid

**Comment by surapunoyousei** [comment]
Flatpak ?

**Comment by Jeefongithub** [comment]
> Flatpak ?

deb package from https://ppa.floorp.app

**Comment by surapunoyousei** [comment]
I do not believe it is on the browser's side. I don't think it's the browser, because there are a lot of Linux users out there, and there is nothing else to report.

Try the Tarball version

**Comment by Jeefongithub** [comment]
Downloaded tarball, copied profile over and can confirm issue is still present. Just to test i downloaded latest firefox version, copied over profile, and can confirm issue does not persist

**Comment by surapunoyousei** [comment]
Why are you copying profiles?

**Comment by surapunoyousei** [comment]
In my env (WSL2 Ubuntu) can change time zone
![image](https://github.com/Floorp-Projects/Floorp/assets/73892113/033486a4-997b-40de-923d-d6f1fc3341cf)

**Comment by surapunoyousei** [comment]
Or... Use addon
https://addons.mozilla.org/ja/firefox/addon/change-timezone-time-shift/

**Comment by Jeefongithub** [comment]
> Why are you copying profiles?

I always copy profiles when migrating between firefox forks. I can test without

> In my env (WSL2 Ubuntu) can change time zone ![image](https://private-user-images.githubusercontent.com/73892113/307388804-033486a4-997b-40de-923d-d6f1fc3341cf.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDg3MDY0OTUsIm5iZiI6MTcwODcwNjE5NSwicGF0aCI6Ii83Mzg5MjExMy8zMDczODg4MDQtMDMzNDg2YTQtOTk3Yi00MGRlLTkyM2QtZDZmMWZjMzM0MWNmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMjMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjIzVDE2MzYzNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE4NDcyZGM4YTZlOGRjNWJmNTA0NzFlZThiMGM0Y2NhNmM3MmFjY2Q1NWRlZTIyYTEyYzI4NjNmNGJlNmU1NTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.kP4V1B5GikLZ9p0iLVB9-N0YGMBA_CfVGLOVn6EreEQ)

i don't have/see that option

> Or... Use addon https://addons.mozilla.org/ja/firefox/addon/change-timezone-time-shift/

I'll test the addon

**Comment by surapunoyousei** [comment]
Try a new profile.

**Comment by surapunoyousei** [comment]
If you have any problems, I can help you with some data migration.

Feel free to post them.
